tests/components/synology_dsm/test_config_flow.py

```diff
@@ -70,10 +72,10 @@ def mock_controller_service_failed():
     with patch(
         "homeassistant.components.synology_dsm.config_flow.SynologyDSM"
     ) as service_mock:
-        service_mock.return_value.login = Mock(return_value=True)
-        service_mock.return_value.information = Mock(serial=None)
-        service_mock.return_value.utilisation = Mock(cpu_user_load=None)
-        service_mock.return_value.storage = Mock(disks_ids=None, volumes_ids=None)
+        service_mock.return_value.information.serial = None
```

Comment:
```suggestion
        service_mock.return_value.login = True
        service_mock.return_value.information.serial = None
```
?

Reply:
login is a callable and not a property so I don't think this will work

Reply:
I've edited it to be a callable